week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -6,10 +6,21 @@
   Create a function that returns an array of colonisers that will stay, according to the above rules.
 
   NOTE: don't include any element that is not a "family".
+  Hint: whenever you read the above the instructions, try to come up with the main input and output and logic
+  Input is an array
+  Output is an array
+  logic only strings that start with A, and finish with family
+  
 */
 
-function colonisers() {
+// function includeFamily(lastName) {
+//   return lastName.startsWith("A");
+// }
 
+function colonisers(voyager) {
+  return voyager.filter(
+    (name) => name.charAt(0) === "A" && name.includes("family")
```

Comment:
Also check out using `.startsWith()` too:
```suggestion
    (name) => name.startsWith('A') && name.includes("family")
```
Your way works too - it is just good to be aware of alternative solutions too :)